code/modules/mob/living/carbon/human/human.dm

```diff
@@ -408,21 +408,34 @@
 /mob/living/carbon/human/proc/get_idcard()
 	if(wear_id)
 		return wear_id.GetID()

```

Comment:
And here. Compact code makes me feel claustrophobic and is generally harder to read.